sources/engine/Stride.Assets/SpriteFont/SystemFontProvider.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
 // Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
-
 using System.Linq;
 using SharpDX.DirectWrite;
 using Stride.Core.Assets.Compiler;
 using Stride.Core;
 using Stride.Core.Diagnostics;
 using Stride.Assets.SpriteFont.Compiler;
 using Stride.Graphics.Font;
+using System;
```

Comment:
Nit: sort usings

Reply:
done!

Reply:
Usually. System usings are first and then the rest in dictionary order. But that's ok here.

We might even use implicit usings later which would hide those.